playbooks/templates/traefik.hcl.j2

```diff
@@ -125,7 +125,9 @@ entryPoint = "traefik"
       email =  "{{ job_fact.certificate_email }}"
       storage = "/config/{{ nomad_datacenter }}.json"
       # comment for production
+      {% if gitlab_runners_enabled is false %}
       caServer = "https://acme-staging-v02.api.letsencrypt.org/directory"
```

Comment:
Why only when gitlab_runners_enabled? This makes no sense.
Create a letsencrypt caServer variable where you put both, and depending on a prd/dev environment you select which one you want to test with.

Reply:
Good! That seems like a better solution.